gpmp2/kinematics/ForwardKinematics-inl.h

```diff
@@ -35,7 +35,7 @@ gtsam::Matrix ForwardKinematics<POSE, VELOCITY>::forwardKinematicsPosition(
   forwardKinematics(jp, {}, jpx, nullptr);
 
   // convert vector in matrix
-  gtsam::Matrix jpx_mat(6, nr_links_);
+  gtsam::Matrix jpx_mat(3, nr_links_);
```

Comment:
Any reason this was changed to 3?

Reply:
Yeah this was changes to 3 because it only determine the position (not attitude) in the workspace. If you look at line 39 below it, you can clear see it.